development/stablediffusion-pipelines-cpp/stablediffusion-pipelines-cpp.info

```diff
@@ -0,0 +1,10 @@
+PRGNAM="stablediffusion-pipelines-cpp"
+VERSION="0.1"
+HOMEPAGE="https://github.com/intel/stablediffusion-pipelines-cpp"
+DOWNLOAD="https://github.com/intel/stablediffusion-pipelines-cpp/archive/v0.1/stablediffusion-pipelines-cpp-0.1.tar.gz"
+MD5SUM="b3acd91b7202a83aeaa575c6a1dc6036"
+DOWNLOAD_x86_64=""
+MD5SUM_x86_64=""
+REQUIRES="openvino libtorch-cxx11-abi-shared-with-deps"
```

Comment:
https://slackbuilds.org/repository/15.0/libraries/libtorch-cxx11-abi-shared-with-deps/?search=libtorch-cxx11-abi-shared-with-deps

Is actually a repackaging of x86_64 only binaries. That package should be x86_64 only and this one too.

Reply:
ok, i will update libtorch

Reply:
done in c8c499e2de